docs/v6_upgrade.md

```diff
@@ -11,105 +11,119 @@ Webpacker used to configure Webpack indirectly, which lead to a [complicated sec
 This means you have to configure integration with frameworks yourself, but webpack-merge helps with this. See this example for [Vue](https://github.com/rails/webpacker#other-frameworks) and scroll to the bottom for [more examples](#examples-of-v5-to-v6).
 
 ## How to upgrade to Webpacker v6 from v5
-
-1. Consider changing from the v5 default for `source_entry_path`.
-  ```yml
-    source_path: app/javascript
-    source_entry_path: packs
-  ```
-  consider changing to the v6 default:
-  ```yml
-    source_path: app/javascript
-    source_entry_path: /
-  ```
-  Then consider moving your `app/javascript/packs/*` (including `application.js`) to `app/javascript/` and updating the configuration file. 
+1. Ensure you have a clean working git branch. You will be overwriting all your files and reverting the changes that you don't want.
+
+1. Upgrade the Webpacker Ruby gem and the NPM package
```

Comment:
```suggestion
2. Upgrade the Webpacker Ruby gem and the NPM package
```

Reply:
Using `1.` everywhere results in less merge conflicts! 

@guillaumebriday, @dhh Do you agree that only `1` is better for a numbered list?